client/src/components/date/dashboardCalendar.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import DayPicker from 'react-day-picker';
+
+const DashboardCalendar = ({ subscriptions }) => {
+  const dueDates = [];
+  subscriptions.map((subscription) => {
+    const { dueDate } = subscription;
+    dueDates.push(new Date(dueDate.nextDueDate));
```

Comment:
if this is the only data used you can just destructure directly:

```suggestion
  subscriptions.map(({ dueDate }) => {
    dueDates.push(new Date(dueDate.nextDueDate));
```